test/plugin.js

```diff
@@ -61,7 +61,7 @@ describe('Plugin tests', function() {
     var plugin;
 
     beforeEach(function() {
-      plugin = new Plugin(new Session(new Connection, 'id'), 'name', 'id');
+      plugin = new Plugin(new Session(new Connection('id', {address: 'address'}), 'id'), 'name', 'id');
```

Comment:
Well. A simple mock `sinon.createStubInstance(Connection)` won't work. We need a more complex mock:
```js
var connectionMock = sinon.createStubInstance(Connection);
connectionMock.getOptions.returns({});
```
Cause this form is longer to write than `new Connection('id', {address: 'address'})`, I used the latter. Please confirm once again that I should use `connectionMock`